packages/fuels-test-helpers/src/node_types.rs

```diff
@@ -70,10 +72,10 @@ impl Config {
     pub fn local_node() -> Self {
```

Comment:
`local_node()` and `default()` lead to the same config, we can deduplicate a little